solc/CommandLineInterface.cpp

```diff
@@ -605,36 +605,36 @@ bool CommandLineInterface::readInputFilesAndConfigureRemappings()
 				addStdin = true;
 			else
 			{
-				auto infile = boost::filesystem::path(path);
-				if (!boost::filesystem::exists(infile))
+				boost::filesystem::path inputFilePath = path;
```

Comment:
Would it make sense marking this const?

Reply:
From what I have seen we don't usually make local objects `const` unless there's a good reason. This would make more sense if we were just making everything `const` by default :)

Anyway, I can add it, I'd prefer seeing `const` in more places myself, this just seems like a random place to do it though.